HW1.py

```diff
@@ -0,0 +1,118 @@
+import sys
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from sklearn.ensemble import AdaBoostClassifier, RandomForestClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.tree import DecisionTreeClassifier
+
+
+def main():
+    # import data into pandas df
+    names = ["Sepal Length", "Sepal Width", "Petal Length", "Petal Width", "Class"]
+    df = pd.read_csv("iris.data", names=names)
+    df_norm = pd.melt(df, id_vars=["Class"])
+
+    # print out summary statistics
+    print("Means:")
+    print(np.mean(df[{"Sepal Length", "Sepal Width", "Petal Length", "Petal Width"}]))
+    print("Mins")
+    print(np.min(df[{"Sepal Length", "Sepal Width", "Petal Length", "Petal Width"}]))
+    print("Maxes")
+    print(np.max(df[{"Sepal Length", "Sepal Width", "Petal Length", "Petal Width"}]))
+    print("25th percentiles:")
+    print(df.quantile(0.25, numeric_only=True))
+    print("50th percentiles:")
+    print(df.quantile(0.50, numeric_only=True))
+    print("75th percentiles:")
+    print(df.quantile(0.75, numeric_only=True))
+
```

Comment:
Instead of plotting multiple scatter plots, maybe try using Plotly's scatter matrix

Reply:
I started out with the basic scatters but I've got a scatter matrix a bit further down.